### PR TITLE
chore: Added minimap to gh pages

### PR DESCRIPTION
**Pull request by cesarades**
**Description**
Added the minimap plugin to the plugins gh page. I also added the advanced playground to the minimap demo so that it looks the same as the other plugins.

**Tests**
Ran the page locally and made sure the link and image were working.

### CODE REVIEW

File: examples/blockly-react/package-lock.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
     "": {
       "name": "blockly-react-sample",
       "version": "0.0.0",
-      "hasInstallScript": true,
```
Comment (BeksOmega) [inline examples/blockly-react/package-lock.json]: Can you revert this change?

**Review by BeksOmega — APPROVED** [review]
One fix then LGTM =)

**Review by gonfunko — APPROVED** [review]
